lib/zinnia.js

```diff
@@ -228,7 +245,7 @@ const waitForStdioClose = async ({ childProcesses, controller }) => {
   const results = await Promise.all(
     childProcesses.map(childProcess => once(childProcess, 'close'))
   )
-  const codes = results.map(([code]) => code ?? '<no code>')
+  const codes = results.map(([code, signal]) => code || signal || '<no code>')
```

Comment:
```suggestion
  const codes = results.map(([code, signal]) => code || `<${signal}>` || '<no code>')
```

I think through this the log line will be clearer, as a signal is a special "code", like "no code"

Reply:
Makes sense 👍🏻